CoreHistogramView/src/au/gov/asd/tac/constellation/views/histogram/HistogramDisplay.java

```diff
@@ -543,66 +540,56 @@ private void copySelectedToClipboard(boolean includeCounts) {
     
     @Override
     public void mouseClicked(MouseEvent e) {
-        if (binCollection != null) {
-            if (e.getButton() == MouseEvent.BUTTON3) {
-                copyMenu.show(this, e.getX(), e.getY());
-            }
+        if (binCollection != null &&
+            e.getButton() == MouseEvent.BUTTON3) {
```

Comment:
Minor but I think we prefer `&&` at the start of the newline rather than the end of the old line